modules/tools/00_mod.mk

```diff
@@ -132,15 +132,16 @@ tools += gci=v0.13.4
 tools += yamlfmt=v0.12.1
 
 # https://pkg.go.dev/k8s.io/code-generator/cmd?tab=versions
-K8S_CODEGEN_VERSION := v0.29.3
+K8S_CODEGEN_VERSION := v0.30.1
 tools += client-gen=$(K8S_CODEGEN_VERSION)
 tools += deepcopy-gen=$(K8S_CODEGEN_VERSION)
 tools += informer-gen=$(K8S_CODEGEN_VERSION)
 tools += lister-gen=$(K8S_CODEGEN_VERSION)
 tools += applyconfiguration-gen=$(K8S_CODEGEN_VERSION)
-tools += openapi-gen=$(K8S_CODEGEN_VERSION)
 tools += defaulter-gen=$(K8S_CODEGEN_VERSION)
 tools += conversion-gen=$(K8S_CODEGEN_VERSION)
+# https://github.com/kubernetes/kube-openapi
+tools += openapi-gen=f0e62f92d13f418e2732b21c952fd17cab771c75
```

Comment:
see https://github.com/kubernetes/kubernetes/commit/1a4450f6ffdad44bfc12fae5911c93e3bd238f0f

Reply:
Confirmed that the SHA matches master at the time of writing: https://github.com/kubernetes/kube-openapi/tree/f0e62f92d13f418e2732b21c952fd17cab771c75